extensions/wikia/ExactTargetUpdates/maintenance/ExactTargetUpdateUserEditsPerWikiMaintenance.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Maintenance script for updating number of edits made by user on a wiki
+ * in ExactTarget database
+ *
+ * @package MediaWiki
+ * @addtopackage maintenance
+ *
+ * @author Kamil Koterba <kamil@wikia-inc.com>
+ *
+ */
+
+require_once( __DIR__ . "/../../../../maintenance/commandLine.inc" );
```

Comment:
I'd switch this to be a standard maintenance script using Maintenance.php, i.e.:

```php
require_once __DIR__ . '/../../../../maintenance/Maintenance.php';

class ExactTargetUpdateUserEditsPerWikiMaintenance extends Maintenance {
...
}


$maintClass = 'ExactTargetUpdateUserEditsPerWikiMaintenance';
require_once RUN_MAINTENANCE_IF_MAIN;
```

commandLine.inc is technically considered a wrapper for old-style maintenance scripts and since you're already using a class, makes the change easy.